src/store/model/coolblue.ts

```diff
@@ -0,0 +1,72 @@
+import {Store} from './store';
+
+export const Coolblue: Store = {
+	labels: {
+		inStock: {
+			container: '.product-order',
+			text: ['bestel snel', 'Morgen in huis']
```

Comment:
You'll want to lowercase these as we do comparisons with all lowercase.
```suggestion
			text: ['bestel snel', 'morgen in huis']
```